constraints-dev.txt

```diff
@@ -12,4 +12,6 @@
 # botocore==1.20.106 is compatible with boto3==1.17.106
 botocore==1.20.106  # From aiobotocore v1.4.0 dependencies https://pypi.org/project/aiobotocore/
 boto3==1.17.106  # from botocore==1.20.106 dependency
+
+asttokens==2.0.5 # Transitive dependency of IPython that has an import error - https://github.com/gristlabs/asttokens/issues/85
```

Comment:
Instead of pinning let's do `asttokens!=2.0.6`. Looks like this this github issue was just closed and they rereleased this package: https://pypi.org/project/asttokens/#history